.github/workflows/commit-message-check.yaml

```diff
@@ -0,0 +1,99 @@
+name: Commit Message Check
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+env:
+  error_msg: |+
+    See the document below for help on formatting commits for the project.
+
+    https://github.com/confidential-containers/community/blob/main/CONTRIBUTING.md#patch-format
+
+jobs:
+  commit-message-check:
+    runs-on: ubuntu-latest
+    name: Commit Message Check
+    steps:
+      - name: Get PR Commits
+        if: ${{ !contains(github.event.pull_request.labels.*.name, 'force-skip-ci') }}
+        id: "get-pr-commits"
+        uses: tim-actions/get-pr-commits@v1.2.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          # Filter out revert commits
+          # The format of a revert commit is as follows:
+          #
+          # Revert "<original-subject-line>"
+          #
+          filter_out_pattern: '^Revert "'
+
+      - name: DCO Check
```

Comment:
We are already running the DCO checking with another action. For example, https://github.com/confidential-containers/cloud-api-adaptor/pull/46/checks?check_run_id=6684767394 . I suggest to keep only the DCO check implemented in this worklof.